### PR TITLE
Add dispatch events to run actions

### PR DESCRIPTION
**Pull request by stevemessick**
The `pull_request_target` should allow actions to run as expected on repo forks. The `workflow_dispatch` is supposed to enable manual triggering from the github UI.

### CODE REVIEW

File: src/io/flutter/module/settings/PlatformsForm.kt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package io.flutter.module.settings
```
Comment (helin24) [inline src/io/flutter/module/settings/PlatformsForm.kt:1]: is this related?

**Review by helin24 — APPROVED** [review]

File: src/io/flutter/module/settings/PlatformsForm.kt (hunk 1 above)
Comment (stevemessick) [reply src/io/flutter/module/settings/PlatformsForm.kt:1]: I don't know how that got included.
